applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot_codec.c

```diff
@@ -68,7 +68,7 @@ static int static_modem_data_add(cJSON *parent,
 				 struct cloud_data_modem_static *data)
 {
 	int err = 0;
-	char nw_mode[50];
+	char nw_mode[50] = {0};
```

Comment:
I wonder why in this case the string is initialized, but above not:

```c
	char imei_buf[20];
```

Reply:
`nw_mode` must be initialized because it's used in a string concatenation on line 104, so random data in the buffer may affect that if none of the if clauses on line 97 and 99 evaluate to true.
'imei_buf` doesn't need to be initialized because it will be populated by `at_cmd_write()` regardless of content, so there's no risk in the `strcpy()` on line 178  that `imei_buf` is unitialized